### DISCUSSION TITLE
AntD Upgrade Documentation

### DISCUSSION THREAD

**Discussion by dleh-wss** [ant-design/ant-design#46932] [General]
Hey all,

Just wanted to open up a feedback post revolving around the upgrade from AntD v4 to v5. When doing an upgrade, we saw that DayJS was being used instead of moment, which I am all for as DayJS is a more lightweight application as we had found. 

My gripe with this was there was no indication that the functions in the moment package, could be moved to a plugin system in DayJS. There was quite a few times where things were not working as expected and we found that it was because of a plugin that we had not imported. We were unaware of the plugins that were coming, and we were hoping that bigger differences like that would be explained in future upgrade documentation.

Here is the documentation that I am referencing: https://ant.design/docs/react/migration-v5

**Comment by zombieJ** [comment]
Let me add note about this. Developer should notice the plugin difference with day.js
